.github/workflows/build.yaml

```
@@ -0,0 +1,49 @@
+name: Build and verify
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - jdk: 11.0.16
+            sonarEnabled: true
+          - jdk: 17.0.4
+            sonarEnabled: false
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.0.2
+        with:
+          # Fetch entire history for SonarCloud analysis.
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v3.4.1
```

```suggestion
        uses: actions/setup-java@v3.5.1
```
A newer version is out :)